Essentials/src/main/resources/config.yml

```diff
@@ -951,66 +946,65 @@ chat:
   # {TEAMNAME} - The sender's scoreboard team name.
   # {TEAMPREFIX} - The sender's scoreboard team prefix.
   # {TEAMSUFFIX} - The sender's scoreboard team suffix.
+  #
+  # Note: The {DISPLAYNAME} placeholder includes {PREFIX} and {SUFFIX} by default.
+  # Using these together may result in double prefixes/suffixes in chat.
 
   format: '<{DISPLAYNAME}> {MESSAGE}'
   #format: '&7[{GROUP}]&r {DISPLAYNAME}&7:&r {MESSAGE}'
-  #format: '&7{PREFIX}&r {DISPLAYNAME}&r &7{SUFFIX}&r: {MESSAGE}'
+  #format: '&7{PREFIX}&r {NICKNAME}&r &7{SUFFIX}&r: {MESSAGE}'
 
   # You can also specify a format for each type of chat.
   #format:
   #  normal: '{WORLDNAME} {DISPLAYNAME}&7:&r {MESSAGE}'
   #  question: '{WORLDNAME} &4{DISPLAYNAME}&7:&r {MESSAGE}'
   #  shout: '{WORLDNAME} &c[{GROUP}]&r &4{DISPLAYNAME}&7:&c {MESSAGE}'
 
-  # You can specify a format for each group.
+  # You can also specify a format for each group.
+  # If using group formats, remove the # to activate the setting.
+  # Note: Group names are case-sensitive, so you must match them up with your permissions plugin.
```

Comment:
Possibly worth a note on LuckPerms group aliases - aliases change the Vault group name